client/src/components/routing/Routes.js

```diff
@@ -28,6 +29,7 @@ const Routes = () => {
         <Route exact path="/adminlogin" component={AdminLogin} />
         <Route exact path="/login" component={Login} />
         <Route exact path="/admineditprofile" component={AdminEditProfile} />
+        <Route exact path="/usereditprofile" component={UserEditProfile} />
```

Comment:
You may want to make this route as a `PrivateRoute` (means only logged in user has access to this route), the reason `/admineditprofile` is not a PrivateRoute is because it is not ready yet.